cmd/vic-machine/create/create.go

```diff
@@ -1160,9 +1157,66 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 		return err
 	}
 
+	// vic-init will try to reach out to the vCenter/ESXi host.
+	log.Info("Checking connectivity with the target vCenter/ESXi")
+	vch, err := executor.NewVCHFromComputePath(c.Data.ComputeResourcePath, c.Data.DisplayName, validator)
+	if err != nil {
+		executor.CollectDiagnosticLogs()
+		log.Errorf("Failed to get Virtual Container Host %s", c.Data.DisplayName)
+		log.Error(err)
+		return errors.New("Running diagnostics failed.")
+	}
+
+	// Checking if VC/ESXi can respond to ping request.
+	const pingTestTxt = "VC/ESXi API Ping Test:"
+	cd, err := executor.CheckVCPingFromAppliance(ctx, vch, vchConfig.Target)
```

Comment:
Don't put a ping test in here - there are plenty of people who will be blocking ICMP packets.
The only thing we care about is whether the endpointVM can talk to vSphere API - just the HEAD request will suffice.

Ping is solely for diagnostics in the failure case.